### ISSUE TITLE
The plugin still doesn't work for Sketch 3.3-There is a way may work

### ISSUE THREAD

**Issue by Burbur2088** [getflourish/Sketch-Style-Inventory#45]
I downloaded it  and install again. It still doesn't work for version 3.3

Is it possible to fix or did I do anything wrong?

Thank you!

**Comment by frankko** [comment]
I don’t know if this is the same problem, but I can confirm issues with some of the commands.

Inventory → Color Inventory and Inventory → Generate don’t work with the same error:

`
4/17/15 2:56:59.241 PM Color Inventory (Sketch Plugin)[868]: TypeError: undefined is not a function (evaluating 'hexColors.count()’)
`
`
line: 716
`
`
sourceURL: ~/Library/Application Support/com.bohemiancoding.sketch3/Plugins/Sketch Style Inventory/Inventory/Color Inventory.sketchplugin
`
`
column: 32
`

**Comment by faisalchaudryuk** [comment]
Experiencing the same issue as well.

**Comment by fabnoe** [comment]
Some here

**Comment by Burbur2088** [comment]
Hey guys,

I don't know if it is helpful but I kind of tried a different way to use the plugin. It works for me.

1. Delete all previous versions of this plugin from your computer, and from your plugin folder.
2. Download again from the link in the post.
3. Put the new one in your plugin folder.

The color inventory works fine for me after I did the steps above.

For the text inventory, I think if your file is located too deep in the file system, it doesn't save the generated text inventory. 

I surfaced my file, and I generate text inventory again. It works. 

Hope it will help you too! :-)

/Burbur
